README.md

```diff
@@ -0,0 +1,11 @@
+# kyuubi
+
+This is a Kubernetes charm for [Apache Kyuubi](https://kyuubi.apache.org/).
+
+Charmhub package name: kyuubi
```

Comment:
same here

Reply:
I'm more inclined to use `kyuubi-k8s` as the name, following the similar pattern we do for `spark-history-server-k8s`. I'll update this accordingly.